tests/unit/test_id_cli.py

```diff
@@ -40,16 +40,29 @@ def mock_exists(mocker):
     mock_exists.return_value = True
 
 
-@pytest.fixture(scope='module')
-def abc_src_dir(tmp_path_factory):
+def _abc_src_dir(tmp_path):
     """Src dir containing three workflows, a, b & c."""
-    cwd_before = Path.cwd()
-    tmp_path = tmp_path_factory.getbasetemp()
     os.chdir(tmp_path)
     for name in ('a', 'b', 'c'):
         Path(tmp_path, name).mkdir()
         Path(tmp_path, name, WorkflowFiles.FLOW_FILE).touch()
-    yield tmp_path
+    return tmp_path
+
+
+@pytest.fixture(scope='module')
+def mod_abc_src_dir(tmp_path_factory):
+    """Module scope _abc_src_dir"""
+    tmp_path = tmp_path_factory.getbasetemp()
+    cwd_before = Path.cwd()
+    yield _abc_src_dir(tmp_path)
+    os.chdir(cwd_before)
+
+
+@pytest.fixture
+def abc_src_dir(tmp_path):
```

Comment:
`test_parse_ids_src_run` would pass when run singly, but fail when run as part of a module. 😢

Reply:
What was the motive behind these changes?

Reply:
They're total garbage.